src/main/webapp/app/course/shared/course-base-container/course-base-container.component.scss

```
@@ -4,77 +4,61 @@
-    .mat-drawer-content {
-        position: sticky;
+    .layout-content {
+        flex: 1;
+        min-width: 0;
+        position: relative;
         z-index: 1;
-        display: block;
 
         @include dynamic-content-height(--spacing-modules, --system-notification-height);
-        width: calc(100vw - $menu-width-open);
         overflow: hidden;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)